Selcuk- 8 HAfta

slckmndl opened the pull request:
Farkli bir kampa transfer ve ayni zamanda hastalik gecirdigimden odevlerden iki tanesi eksik..

slckmndl committed:
Selcuk 8 HAfta

dadasyvz commented on 1- Asal SAyi.py:
```diff
@@ -0,0 +1,16 @@
+    if Asalsayiislemi(sayi):
+        print(sayi, "Sayisi Asal Sayıdır")
+    else:
+        print(sayi, "Sayisi Asal Değildir")
```
*yazilan fonksiyon dogru ciktilar vermektedir

dadasyvz commented on 11 - Ozgun Liste.py:
```diff
@@ -0,0 +1,16 @@
+
+
+
+print(Listeayiklama())
```
**soruda verilen bir listenin içindeki özgün elemanları ayırıp yeni bir liste olarak veren bir fonksiyon yazilmasi istenmistir. Bu herhangi bir kullanici girisi olabilmeli, input alinan bir fonksiyon olarak tasarlanmalidir.
*liste olarak girilen bir objenin split edilerek setlenmesi ile daha kolay sonuca ulasilacagi hatirlatmasi ile duzeltmeniz onerilir.

slckmndl replied:
Guncellendi..Input Eklendi..

dadasyvz commented on 12 - Ters Kontrol.py:
```diff
@@ -0,0 +1,15 @@
+    a = input("KElimeyi Giriniz : ")
+
+    print(terskontrol(a))
+
```
** soruda verilen inputların tersten de aynı olup olmadığını kontrol eden bir fonksiyon yazınız. örnek: madam, taco cat, utrecht sonuç: True, True, False
*KElimeyi Giriniz : selim asa madam 
True
None
*kullanicinin istedigi kadar kelime girisini ve kontrollerini saglayan bir fonksiyon tasarlamalisiniz

dadasyvz commented on 2 - Tam Bolen.py:
```diff
@@ -0,0 +1,12 @@
+
+while True:
+    a = int(input("Sayıyi Giriniz : "))
+    print(a, "Sayisinin Tam Bolenleri :", TambolumIslemi(a),"\n")
```
*yazilan fonksiyon dogru ciktilar vermektedir

dadasyvz commented on 3- Mukemmel Sayi.py:
```diff
@@ -0,0 +1,16 @@
+    if mukemmelsayislemi(i):
+        m_s_listesi.append(i)  # listeye ekledik.
+
+print("Mükemmel sayılar : ", m_s_listesi)
```
*yazilan fonksiyon dogru ciktilar vermektedir

dadasyvz commented on 4 ve 5 - Obeb Okek.py:
```diff
@@ -0,0 +1,18 @@
+
+
+
+print(okek(2,4))
```
*yazilan fonksiyon dogru ciktilar vermektedir

dadasyvz commented on 6- Sayi okunus.py:
```diff
@@ -0,0 +1,101 @@
+
+
+
+
```
*yazilan fonksiyon dogru ciktilar vermektedir

dadasyvz commented on 8 - Faktoriye Hesaplama.py:
```diff
@@ -0,0 +1,16 @@
+
+    for i in range(2, sayi + 1):
+        faktoriyel *= i
+        return faktoriyel
```
*return bir tab geride olacak sekilde duzenlenirse dogru sonuclar vermektedir

dadasyvz commented on 9 - Buyuk Kucuk HArf.py:
```diff
@@ -0,0 +1,18 @@
+    return kucuktoplam,buyuktoplam
+while True:
+    a = input("Lutfen Kelimeyi Giriniz : ")
+    print("Buyuk Ve Kucuk HArfler : ",harfsayisi(a),"\n")
```
**sorumuz kullanıcıdan bir input alan ve bu inputun içindeki büyük ve küçük harf sayılarının veren bir fonksiyonu yazmaniz seklindedir.Buna yonelik duzeltme yapmaniz onerilir

dadasyvz replied:
**kod bu haliyle bosluk ve diger alfabetik olmayan karakterleri buyuk olarak eklemekte ve hatali ciktilar vermektedir

slckmndl committed:
Update 9 - Buyuk Kucuk HArf.py

slckmndl committed:
Update 11 - Ozgun Liste.py

slckmndl committed:
Update 11 - Ozgun Liste.py

Input Eklendi..

slckmndl committed:
Update 12 - Ters Kontrol.py

Guncellendi..

slckmndl committed:
Update 8 - Faktoriye Hesaplama.py

Guncellendi..

slckmndl committed:
Update 9 - Buyuk Kucuk HArf.py

Guncellendi..Buyuk ve kucuk harf sayilarini veriyor..

slckmndl committed:
Create 7. pisagor ucgeni

Eklendi

slckmndl committed:
Create 10 - tire isareti ile ayrilan kelimeler

eklendi

slckmndl commented:
Eksikler yuklendi ve yanlislar duzeltildi..

dadasyvz commented:
12. sorunun duzeltilmesi ile tamamlanmis olacaktir

slckmndl commented:
Hocam 12. Soruyu guncellemistim zaten ama ? Bir karisiklik mi oldu

4 Eyl 2019 Çar, saat 14:30 tarihinde yavuz koksal <notifications@github.com>
şunu yazdı:

>
>    1. sorunun duzeltilmesi ile tamamlanmis olacaktir
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pycoders-nl/8.hafta_odevler-Fonksiyonlar/pull/9?email_source=notifications&email_token=AMI7J35TTQK3FIW4D5SOSGTQH6S75A5CNFSM4IKMNC6KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD53MZXA#issuecomment-527879388>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMI7J35GMTFN6ORUKFR34HLQH6S75ANCNFSM4IKMNC6A>
> .
>

slckmndl committed:
Update 9 - Buyuk Kucuk HArf.py

Artik ozel karakter ve bosluklari buyuk listesine eklemiyor ve gormezden gelip uyari vererek buyuk ve kucuk harf sayilarini veriyor..

slckmndl committed:
Update 12 - Ters Kontrol.py

Sanirim gormediniz, sonunda kelimeleri ters cevirip True True False olarakta sonuc veriyor..Ben sona bir print ekleyip guncelledim..SAygilarimla..

dadasyvz commented:
**duzeltmeleriniz neticesinde tum kodlariniz eksiksiz calsimaktadir....
